common/autoware_ad_api_msgs/srv/InterfaceVersion.srv

```diff
@@ -0,0 +1,6 @@
+---
+autoware_ad_api_msgs/ResponseStatus status
+string version
```

Comment:
What is this `version` for?
It seems to have duplicated information with `major.minor.patch`.

Reply:
It was just for convenience, so I removed the redundant string. https://github.com/autowarefoundation/autoware.universe/commit/7f1b1e2c28f792c3cdc273f9cd45dc85bff9a0a7